### DISCUSSION TITLE
Scrapper

### DISCUSSION THREAD

**Discussion by mananjoshi2000** [WebMetadataRetrieval/15-Web_Metadata_Retrieval#40] [General]
What is the status of the Scrapper?

**Comment by mananjoshi2000** [comment]
I have found a python library to extract metadata from the 'meta' tags of the web pages.
https://pypi.org/project/metadata-parser/

Check it out!

**Reply by devsoftzz** [reply]
Looks Interesting. Can reduce lot of LOC and efforts

**Comment by devsoftzz** [comment]
> What is the status of the Scrapper?

@Paresh501 @Isha-Desai have pushed code to scrape webpages of Wikipedia, Flipkart and scrapper team is working on it and will try to cover atleast 10 website which are most popular

**Reply by Isha-Desai** [reply]
We have planned on covering some websites and so are still working on them.

**Comment by Isha-Desai** [comment]
Yes, I have finished making the scrapper for the Flipkart website.

**Comment by mananjoshi2000** [comment]
I have added a general scrapper. You guys may check.
